modules/maven/build-locally.sh

```diff
@@ -105,6 +105,18 @@ function check_secrets {
     success "secrets baseline timestamp content has been removed ok"
 }
 
+function check_if_script_invoked_directly() {
```

Comment:
As this is used everywhere, why not just call the function in the main tools folder from everywhere ?